programs/stake/src/stake_state.rs

```diff
@@ -3450,6 +3454,33 @@ mod tests {
                 true,
             )
         );
+
+        // credis_observed isn't reset for stale vote accounts while no rewards
```

Comment:
```suggestion
        // credits_observed isn't reset for stale vote accounts while no rewards
```

Reply:
Here's a suggestion for this comment (which I found confusing):
`credits_observed remains at previous level when vote_state credits are not advancing and inflation is disabled`

Reply:
admittedly, I had some difficulty to describe this.. Thank for helping writing. :)